prometheus/prometheus.yaml

```
@@ -0,0 +1,10 @@
+global:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)